tests/test_game_time.py

```diff
@@ -60,3 +62,35 @@ def ow_player_data_response_not_found():
     ctx = await pg.prepare_game_times_context([OW_ID])
     result = await pg.get_game_time(OW_ID, ctx)
     assert result == GameTime(OW_ID, minutes, None)
+
+
+@pytest.mark.asyncio
+async def test_last_played_when_unknown_game_in_config(pg, config_parser):
+    game = Blizzard['diablo3']
+    unknown_game_uid = 'unknown_game'
+    with pytest.raises(KeyError):
+        assert Blizzard[unknown_game_uid]  # test precondition
+
+    # this way we fill mocks with desired data we want to test; the trick is pg fixture uses
+    # config_parser internally as part of local_client, but it can be also:
+    # pg.local_client.config_parser.games = ... (but sometimes it's not possible)
```

Comment:
can be removed. It was just for you to explain

Reply:
ok done.